src/globals/scss/_theme-tokens.scss

```diff
@@ -173,6 +173,8 @@
   $data-table-heading-transform: uppercase !default !global;
   $data-table-heading-border-bottom: 1px solid $brand-01 !default !global;
   $data-table-row-height: 2rem !default !global;
+  $data-table-zebra-color: #fcfcfc !default !global;
+  $data-table-column-hover: rgba(#000000, 0.1) !default !global;
```

Comment:
Is there a token we could use? Or IDL interaction color?

Reply:
Re: line 177 - This was a mix up with design. The design spec was using this color but turns out it should be a different one that does have a token. Just fixed.